docs/webaverse/silk.md

```diff
@@ -0,0 +1,24 @@
+---
+id: silk 
+title: SILK 
+---
+
+SILK is a token that is used in Webaverse primarily to mint [NFTs](./nfts). It's main purpose is to invite others into the network and prevent spam so together we can build a web of trust as the Webaverse scales.
+
+If you're a Discord server admin that wants to distribute SILK amongst your community to use the [NFT Discord Bot](https://webaverse.com/discordbot) then ping us on [Discord](https://discord.gg/3byWubumSa) or [email us](hello@webaverse.com).
+
+## FAQ
+
+### Why the name SILK?
+
+Backstory: For years we've been thinking about and working on interoperability problems in the space. We want projects to be more compatible with one another so we can have a Metaverse or 3D internet. Right now virtual worlds act as separate colonies or countries, each with their own currencies and moderation practices.
```

Comment:
I'd skip this and just use the second paragraph.

Reply:
If we want to talk about ourselves, we should do it as a footnote to the point that we are really trying to make.

Reply:
gunna leave it out for now